packages/11ty/_includes/components/figure/image/image-tag.js

```diff
@@ -13,8 +13,8 @@ const path = require('path')
 module.exports = function(eleventyConfig) {
   const { imageDir } = eleventyConfig.globalData.config.figures
 
-  return function ({ alt='', src='' }) {
-    const imageSrc = src.startsWith('http') ? src : path.join(imageDir, src)
+  return function ({ alt='', src='', isStatic=false }) {
+    const imageSrc = src.startsWith('http') || isStatic ? src : path.join(imageDir, src)
```

Comment:
I don't love calling this option `isStatic` but I need some kind of flag to omit the `imageDir`, since static outputs (`print-image`, `thumbnail`, and `static-figure-image` all live in `_site/iiif`, rather than `_assets/images`)